templates/userdash/userdash-top.html

```diff
@@ -1,13 +1,62 @@
 {{ define "content" }}
 
-<section class="jumbotron jumbotron-fluid text-center bg-dark">
-	<div class="container">
-		<h1 class="jumbotron-heading text-ca-yellow2"><strong>Dashboard</strong></h1>
+{{ template "basicspotlight" . }}
+
+<section class="container-fluid text-center">
+	<h2 class="font-weight-bold">Account Details</h2>
+
+	<div class="my-4"></div>
+
+	<div id="userdash-profile">
+		<div class="row justify-content-between">
+			<div class="col text-right">Full name:</div>
+			<div class="col text-left">{{ .user.FullName }}</div>
+		</div>
+		<div class="row justify-content-between">
+			<div class="col text-right">Gustavus username:</div>
+			<div class="col text-left">{{ .user.GusUsername }}</div>
+		</div>
+		<div class="row justify-content-between">
+			<div class="col text-right">Gustavus Student ID:</div>
+			<div class="col text-left">{{ .user.GusID }}</div>
+		</div>
+		<div class="row justify-content-between">
+			<div class="col text-right">Total Orders:</div>
+			<div class="col text-left">{{ .totalOrders }}</div>
+		</div>
 	</div>
 </section>
 
-<section class="container">
-	<h3 class="text-center">This page has yet to be implemented</h3>
+<section class="container text-center mt-5">
+	<h2 class="font-weight-bold">Past Orders</h2>
+	{{ if and (.orders) (len .orders) }}
+	<table class="table table-responsive-md">
+		<thead>
+			<tr>
+				<th scope="col">#</th>
+				<th scope="col">Destination</th>
+				<th scope="col">Date</th>
+				<th scope="col">Amount</th>
+				<th scope="col">Status</th>
+				<th scope="col"></th>
+			</tr>
+		</thead>
+		<tbody>
+			{{ range .orders }}
+			<tr>
+				<th scope="row">{{ fromTagToNumber .Tag }}</th>
+				<td>{{ .Destination.Name }}</td>
+				<td>{{ .CreatedAt.Format "Jan _2 3:04pm" }}</td>
```

Comment:
@dpham273 this part `.CreatedAt.Format "Jan _2 3:04pm"` is equivalent in golang: `order.CreatedAt.Format("Jan _2 3:04pm"). explaination:

- CreatedAt has the type `time.Time`
- Golang times can be formatted using the function `Format`. If you want to read about it: https://golang.org/pkg/time/#Time.Format
- The reason it is written this way is because of golang's template syntax: [here](https://golang.org/pkg/text/template/#hdr-Pipelines)